src/CustomTypes/DownloadSongsPlaylistViewController.cpp

```diff
@@ -0,0 +1,65 @@
+#include "CustomTypes/DownloadSongsPlaylistViewController.hpp"
+
+#include "CustomLogger.hpp"
+#include "HMUI/Touchable.hpp"
+#include "HMUI/TableView.hpp"
+#include "UnityEngine/UI/ContentSizeFitter.hpp"
+#include "UnityEngine/UI/LayoutElement.hpp"
+
+#include "playlistcore/shared/CustomTypes/CoverTableCell.hpp"
+#include "bsml/shared/BSML-Lite.hpp"
+
+DEFINE_TYPE(SongDownloader, DownloadSongsPlaylistViewController);
+
+using namespace SongDownloader;
```

Comment:
I don't really like these here as they obscure where the methods are coming from (it confused me already)